### PR TITLE
📝 docs: update close.com source page

### PR DESCRIPTION
**Pull request by ChristoGrab**
## What
Updated and expanded documentation for the Close.com source connector.

## How
- Added link to source
- Added step-by-step instructions for creating API key
- Addressed edge case wherein first-time users have slightly different setup steps
- Expanded setup instructions
- Added example of date formatting
- Removed misleading rate limit metric
- Fixed minor typos

## 🚨 User Impact 🚨
- Easier/improved navigation of Close.com source page

### CODE REVIEW

File: docs/integrations/sources/close-com.md [hunk 1]
```diff
@@ -1,29 +1,51 @@
...
+4. At the top of the page, click **+ New API Key**.
+
+:::caution
+For security purposes, the API Key will only be displayed once upon creation. Be sure to copy and store the key in a secure location.
+:::
+
+For further reading on creating and maintaining Close.com API keys, refer to the 
+[official documentation](https://help.close.com/docs/api-keys-oauth).
+
+### Step 2: Set up the Close.com connector in Airbyte
```
Comment (sherifnada) [inline docs/integrations/sources/close-com.md:26]: I really like this pattern. I think we should update our docs template to match this pattern of 
`step 1: get pre-requisite 1`
`step 2: get pre-requisite 2`
...
`step N: setup the connector in Airbyte`

mind making that change [here](https://hackmd.io/Bz75cgATSbm7DjrAqgl4rw?view)?

File: docs/integrations/sources/close-com.md [hunk 2]
```diff
@@ -1,29 +1,51 @@
...
+
+1. [Log in to your Airbyte Cloud](https://cloud.airbyte.com/workspaces) account, or navigate to the Airbyte Open Source dashboard.
```
Comment (sherifnada) [inline docs/integrations/sources/close-com.md:28]: same comment about collapsing steps 1-4 as in the other PR

File: docs/integrations/sources/close-com.md [hunk 3]
```diff
@@ -1,29 +1,51 @@
...
+2. In the left navigation bar, click **Sources**.
+
+:::tip
+If this is your first time setting up an Airbyte source, skip the next step and proceed to step 4.
+:::
+
+3. In the top-right corner, click **+ New source**.
+4. Find and select **Close.com** from the list of available sources.
+5. Enter a **Source name** of your choosing.
+6. In the **api_key** field, enter your Close.com **API Key**
```
Comment (sherifnada) [inline docs/integrations/sources/close-com.md:38]: similar to the other PRs, can you make an issue or PR to update these to have human readable `title` attributes such as `title: API Key` and `title: Replication Start Date`

**Review by sherifnada — APPROVED** [review]
LGTM if the requested changes are made

File: docs/integrations/sources/close-com.md (hunk 2 above)
Comment (ChristoGrab) [reply docs/integrations/sources/close-com.md:28]: Fixed 👍

**Pull request merged**